zinc/src/sbt-test/source-dependencies/nested-type-params/test

```diff
@@ -0,0 +1,9 @@
+> compile
+
+# change type alias
+$ copy-file changes/Definitions.scala Definitions.scala
+$ copy-file changes/Usage.scala Usage.scala
+
+
+# Both A.scala and B.scala should be recompiled, producing a compile error
```

Comment:
Is this a copy-and-paste error? There's no A.scala and B.scala, and is the "producing a compile error" comment correct?

Reply:
Yep, this is copy paste error.